content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -16,6 +16,7 @@
 "Elephant Shed","Provides ppc64le build and performance testing for Elephant Shed, a bundle of PostgreSQL tools and web UI for easy administration of PostgreSQL servers"
 "Firefox","Ports Mozilla Firefox to POWER8"
 "GCC/Clang","Contributes to improved support of GCC and Clang projects on POWER machines by working on IBM Linux Technology Center items on bountysource.com"
+"gcc-gfortran", "GNU Compiler for the Fortran programming language"
```

Comment:
The first letter should be capitalized, so it fits with the names assigned to all the other projects in this list.

According to the [website](https://gcc.gnu.org/fortran/), "GNU Fortran" might be a better name to use here.